src/FeedbackForm/FeedbackForm.js

```diff
@@ -0,0 +1,268 @@
+import * as React from "react";
+import AddIcon from "@mui/icons-material/Add";
+import Button from "@mui/material/Button";
+import Card from "@mui/material/Card";
+import CardContent from "@mui/material/CardContent";
+import CloseIcon from "@mui/icons-material/Close";
+import Dialog from "@mui/material/Dialog";
+import DialogActions from "@mui/material/DialogActions";
+import DialogContent from "@mui/material/DialogContent";
+import DialogTitle from "@mui/material/DialogTitle";
+import { Fab } from "@mui/material";
+import IconButton from "@mui/material/IconButton";
+import PropTypes from "prop-types";
+import SentimentSatisfiedAltIcon from "@mui/icons-material/SentimentSatisfiedAlt";
+import SentimentVeryDissatisfiedIcon from "@mui/icons-material/SentimentVeryDissatisfied";
+import TextField from "@mui/material/TextField";
+import TextsmsOutlinedIcon from "@mui/icons-material/TextsmsOutlined";
+import Typography from "@mui/material/Typography";
+import { styled } from "@mui/material/styles";
+import { useState } from "react";
+
+// for parent dialog
+const BootstrapDialog = styled(Dialog)(({ theme }) => ({
+  "& .MuiDialogContent-root": {
+    padding: theme.spacing(2)
+  }
+}));
+
+const BootstrapDialogTitle = props => {
+  const { children, onClose, ...other } = props;
+  return (
+    <DialogTitle sx={{ m: 0, p: 2 }} {...other}>
+      {children}
+      {onClose ? (
+        <IconButton
+          aria-label="close"
+          onClick={onClose}
+          sx={{
+            color: theme => theme.palette.grey[500],
+            position: "absolute",
+            right: 8,
+            top: 8
+          }}
+        >
+          <CloseIcon />
+        </IconButton>
+      ) : null}
+    </DialogTitle>
+  );
+};
+
+BootstrapDialogTitle.propTypes = {
+  children: PropTypes.node,
+  onClose: PropTypes.func.isRequired
+};
+
+export default function FeedbackForm({ onSubmit }) {
+  const [openFeedback, setOpenFeedback] = React.useState(false);
+
+  const handleClickOpen = () => {
+    setOpenFeedback(true);
+  };
+  const handleClose = () => {
+    setOpenFeedback(false);
+  };
+
+  // child dialog
```

Comment:
There are few places where comments could be improved in this file as it is quite long. I will make the changes for you so you can see an example for next time.